examples/src/main/kotlin/io.zenoh/ZPut.kt

```diff
@@ -16,8 +16,8 @@ package io.zenoh
 
 import io.zenoh.keyexpr.intoKeyExpr
 import io.zenoh.prelude.SampleKind
-import io.zenoh.publication.CongestionControl
-import io.zenoh.publication.Priority
+import io.zenoh.prelude.CongestionControl
```

Comment:
It was mostly done to avoid circular dependencies between modules (yes, I know that kotlin/java can handle it). Also these types are really not something proper to publication only since they appear in other contexts (especially after this pr)

Reply:
Ack.